src/main/resources/db.changeLog/changeLog-3.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+		http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet  author="bestuser"  id="3">
+        <sql dbms="!h2, mysql"
+     endDelimiter="\nGO"
+     splitStatements="true"
+     stripComments="true">insert into item (name, category_id, description, image_src) values ('item_name1', 1, 'item_desc1', 'image_src1'),
```

Comment:
instead of hardcoding `category_id` value (1) use subselect `(select id from category where name = 'category_name1')`, because ID can be different in local/test/production databases.
for value 2 -> change sql `where name = 'category_name2'`
for value 3 -> change sql `where name = 'category_name3'`